webapp/components/survey/SurveyForm/components/FormPageNavigation/store/state/state.js

```diff
@@ -17,6 +17,7 @@ const keys = {
   expandedFormPageNavigation: 'expandedFormPageNavigation',
   canEditDef: 'canEditDef',
   surveyCycleKey: 'surveyCycleKey',
+  outerPageChildDefs: 'outerPageChildDefs',
```

Comment:
where do you read those? from what I see, the formPageNavigavtion always filters the nodeDef children.
isn't the problem somewhere when removing the nodeDef, i.e. the nodeDef is not removed from the store?

Reply:
The getter was declared on line 61 in this file, the key was missing and I added. 
Also, this getter is used in `arena/webapp/components/survey/SurveyForm/components/FormPageNavigation/FormPageNavigation.js`
line 18

The noDef was removed from the store but the problem was that we are not updating the local state, this was handled in the other file.